variables/_svgs.pcss

```diff
@@ -265,6 +277,18 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Mail
+ * ----------------------------------------------------------------------------- */
+
+@svg-load mail url('../icons/mail.svg') {
+
+	path {
+		stroke: var(--color-icon-active);
+	}
+
```

Comment:
```suggestion
```

Reply:
The rest of the variables have this trailing newline. If I remove the line will be inconsistent.

Reply:
No need to change. Keep it consistent

Reply:
I took another look, the file doesnt have the last spacing. So I am pushign the suggestion from @paulmskim